### PR TITLE
NEXUS-40061 - more config documentation

### PR DESCRIPTION
**Pull request by bobotimi**

### CODE REVIEW

**Review by oleksiirudyk — APPROVED** [review]

**Review by userSeprid — APPROVED** [review]

File: scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml [hunk 1]
```diff
@@ -239,17 +285,23 @@ spec:
...
     | `ingress.additionalRules`                                   |  Additional rules to add to the ingress | null|
-    | `ingress.tls.enabled`                                       |  Whether or not TLS should be enabled for the ingress | "false"|
+    | `ingress.tls.enabled`                                       |  Whether or not TLS should be enabled for the ingress | false|
     | `ingress.tls.secretName`                                    | The name of a Secret object to store TLS secret in | tlsSecretName|
     | `ingress.dockerSubdomain`                                   |  Whether or not to add rules for docker subdomains | false|
     | `ingress.annotations`                                       |  Annotations for the ingress object | null |
     | `license.fileContentsBase64`                                |  Your Base64 encoded Nexus Repository license | nexus-repo-license (must be overridden via Kubernetes secret object which feeds into env variable variables defined in the operator.yaml . See above)|
```
Comment (lisadurant) [inline scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml:297]: This line is confusing me. I get this is the license file - I'm confused by everything from "which feeds into env variable variables defined in the operator.yaml. See above" 

I'm not sure where above you mean that they should see, and the "variable variables" thing is confusing.

File: scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml [hunk 2]
```diff
@@ -239,17 +285,23 @@ spec:
...
     | `license.secretName`                                        |  The name of a secret object to store your Base64 encoded Nexus Repository license in | nexus-repo-license|
+    | `ingress.dockerIngress.name`                                |  The name of an Ingress object which listens for requests to an existing docker repository in Nexus Repository | nexus-docker-ingress|
+    | `ingress.dockerIngress.enabled`                             |  Whether or not to create an ingress for a docker repository | false|
```
Comment (lisadurant) [inline scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml:300]:
```suggestion
    | `ingress.dockerIngress.enabled`                             |  Whether or not to create an Ingress for a docker repository | false|
```

File: scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml [hunk 3]
```diff
@@ -239,17 +285,23 @@ spec:
...
+    | `ingress.dockerIngress.host`                                |  The docker ingress's host | "example.com"|
```
Comment (lisadurant) [inline scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml:301]:
```suggestion
    | `ingress.dockerIngress.host`                                |  The docker Ingress's host | "example.com"|
```

File: scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml [hunk 4]
```diff
@@ -239,17 +285,23 @@ spec:
...
+    | `ingress.dockerIngress.annotations`                         |  Annotations for the docker ingress object | null |
```
Comment (lisadurant) [inline scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml:302]:
```suggestion
    | `ingress.dockerIngress.annotations`                         |  Annotations for the docker Ingress object | null |
```

File: scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml [hunk 5]
```diff
@@ -239,17 +285,23 @@ spec:
...
+    | `ingress.dockerIngress.annotations`                         |  Annotations for the docker ingress object | null |
```
Comment (lisadurant) [inline scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml]:
```suggestion
    | `ingress.dockerIngress.annotations`                         |  Annotations for the docker Ingress object | null |
```

File: scripts/templates/sonatype.com_v1alpha1_nexusrepo_cr.yaml [hunk 6]
```diff
@@ -9,35 +9,35 @@ spec:
...
   nexusData:
     local:
       accessMode: ReadWriteOnce
-      enabled: "true"
+      enabled: true
       pv:
         matchNodes: example-node-1 example-node-2 example-node-3
         name: nexusrepo-local-storage-pv
-        size: 10Gi
+        size: 2Gi
```
Comment (Mykyta) [inline scripts/templates/sonatype.com_v1alpha1_nexusrepo_cr.yaml:35]: is it an intentional change? I see per our [official docs](https://help.sonatype.com/display/NXRM3M/Sonatype+Nexus+Repository+System+Requirements#SonatypeNexusRepositorySystemRequirements-DiskSpace): "When available disk space drops below 4GB the database will switch to read-only mode."

File: scripts/templates/values.yaml [hunk 7]
```diff
@@ -89,15 +89,15 @@ license:
...
     targetPort: 8081
   docker:  #Nodeport Service for Docker Service
     name: nexus-repo-docker-service
-    enabled: "false" #intentionally a string because it's used to configure operator via env variable
+    enabled: false
     type: NodePort
     protocol: TCP
     port: 9090
-    targetPort: 8081
+    targetPort: 9090
```
Comment (Mykyta) [inline scripts/templates/values.yaml:103]: is it correct?

File: scripts/templates/sonatype.com_v1alpha1_nexusrepo_cr.yaml (hunk 6 above)
Comment (bobotimi) [reply scripts/templates/sonatype.com_v1alpha1_nexusrepo_cr.yaml:35]: This is just for storing logs i.e. not database.

File: scripts/templates/values.yaml (hunk 7 above)
Comment (bobotimi) [reply scripts/templates/values.yaml:103]: Yep. It's for docker connector. It's whatever the user specified as the http port if they enable docker http connector. We should probably add a comment as such

**Review by lisadurant — APPROVED** [review]

File: scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml (hunk 1 above)
Comment (bobotimi) [reply scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml]: https://github.com/sonatype/operator-nexus-repository/pull/5/commits/cb1e4341a5c89fe0ab9af8a585395f43e2dea54f

**Review by Mykyta — APPROVED** [review]

**Pull request merged**